src/main/java/io/neonbee/job/JobVerticle.java

```diff
@@ -74,12 +74,20 @@ public String getName() {
     @Override
     public void start() {
         if (!NeonBee.get(getVertx()).getOptions().shouldDisableJobScheduling()) {
+            if (schedule.isPeriodic() && isTemporalAdjusterInvalid()) {
```

Comment:
isScheduleValid()

or

validateSchedule()

so we can add further checks to it later on?

Reply:
done